Assets/Mapzen/VectorData/GeometryContainer.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.Linq;
```

Comment:
`using System.Linq` is needed for the `.Last()` applied to `List<T>` below, but I can and will remove `using System` 👍 .